### PR TITLE
Reagent Mateirals by Banthebantz, conflict fix edition.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: banthebantz
add: A new room has been added to Engi maint containing various machines, this room is engi only
add: The first of note are two reagent sheet machines which can turn solid reagents into reagent ingots
add: The second is the big forge which can make various custom weapons and bullets out of ingots
add: These weapons can apply reagents on hit and have varying stats based on what reagent you use
add: The third machine of note is a reagent material trait manipulator
add: Some reagents have special traits, these can be seen in the aforementioned machine
add: This machine can add additional traits to forged weapons, fill it with a beaker and add a weapon
add: Reagent ingots can also be used to make reagent floors, walls and doors
add: These also have custom effects, when touched, shot at and heated, please be careful ;)
tweak: Solid chems no longer react with turfs when splashed
/:cl:

It should work again now and it has sprites from Fury

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
this needs TGUI rebuilt so I can test the fukken thing.

**Comment by GuyonBroadway** [comment]
IT FUCKING WORKS!!!

File: hippiestation/code/game/machinery/reagent_forge.dm [hunk 1]
```diff
@@ -0,0 +1,193 @@
+/obj/machinery/reagent_forge
+	name = "material forge"
+	desc = "A bulky machine that can smelt practically any material in existence"
```
Comment (praisenarsie) [inline hippiestation/code/game/machinery/reagent_forge.dm]: . at the end of sentence

File: hippiestation/code/game/machinery/reagent_material_manipulator.dm [hunk 2]
```diff
@@ -0,0 +1,212 @@
+/obj/machinery/reagent_material_manipulator
+	name = "material manipulation machine"
+	desc = "A high tech machine that can both analyse material traits and combine material traits with each other"
```
Comment (praisenarsie) [inline hippiestation/code/game/machinery/reagent_material_manipulator.dm]: . at the end of the sentence

File: hippiestation/code/game/machinery/reagent_sheet.dm [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/obj/machinery/reagent_sheet
+	name = "Reagent Refinery"
+	desc = "Smelts and refines solid reagents into ingots- useable by the forge"
```
Comment (praisenarsie) [inline hippiestation/code/game/machinery/reagent_sheet.dm]: . at the end of the sentence

**Comment by praisenarsie** [comment]
there's periods missing at every desc!!! grammur!!

**Comment by JohnGinnane** [comment]
can we merge this

**Comment by praisenarsie** [comment]
no

**Review by praisenarsie — APPROVED** [review]
fine

**Pull request merged**